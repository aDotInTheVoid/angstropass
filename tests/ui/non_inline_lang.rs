#[nanopass::langs]
mod langs {
    mod foo {}
    mod bar;
}

fn main() {}
