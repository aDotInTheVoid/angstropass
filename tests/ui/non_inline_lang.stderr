error[E0658]: non-inline modules in proc macro input are unstable
 --> tests/ui/non_inline_lang.rs:4:5
  |
4 |     mod bar;
  |     ^^^^^^^^
  |
  = note: see issue #54727 <https://github.com/rust-lang/rust/issues/54727> for more information
  = help: add `#![feature(proc_macro_hygiene)]` to the crate attributes to enable

error: expected an inline module
 --> tests/ui/non_inline_lang.rs:4:5
  |
4 |     mod bar;
  |     ^^^^^^^^
